Simplify the mapping config for QGIS server (#8332)
camptocamp/c2cgeoportal: doc/integrator/backend_qgis.rst

We provide a Docker image named ``camptocamp/geomapfish-qgisserver`` with tag pattern:
``gmf<Major GeoMapFish version}-qgis${Major QGIS}``.

> From version 2.7 the config is just made with the ``GEOMAPFISH_ACCESSCONTROL_BASE_URL`` environment

Reviewer: Seems that the others modes are still supported, but documentation already removed.

Author: Yes, you prefer that I put them is a deprecated section?

> variable the contanis the the docker url present in the OGC servers, by default it's et to

Reviewer: ```suggestion
variable which contains the base URL of the OGC servers, by default it's set to
```

> ``QGISSERVER_URL``. And the plugin wikk search the OGC servers that corresponds to this URL.

Reviewer: ```suggestion
``QGISSERVER_URL``. And the plugin will search for the OGC servers that match with this base URL.
```
